org.wxhexeditor.wxHexEditor.json

```diff
@@ -0,0 +1,73 @@
+{
+    "app-id": "org.wxhexeditor.wxHexEditor",
+    "only-arches": ["x86_64", "i386"],
```

Comment:
As far as I know, it have to be done in a separate file (`flathub.json`), like here:
https://github.com/flathub/com.xnview.XnViewMP/blob/f605b9cf96769bfdf06bf74d3a97f271550067b4/flathub.json